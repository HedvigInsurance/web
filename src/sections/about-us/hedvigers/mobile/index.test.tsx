import { mount } from 'enzyme'
import * as React from 'react'

import { Mobile } from './'

interface Global {
  innerWidth: number
  innterHeight: number
}

declare var global: Global

it('renders correctly', () => {
  const teamtailorUsers = [
    {
      name: 'mock',
      title: 'mock',
      picture: {
        large: 'https://mockimage.com',
      },
    },
  ]

  global.innerWidth = 600

  const wrapper = mount(
    <Mobile title="mock title" teamtailorUsers={teamtailorUsers} />,
  )
  expect(wrapper).toMatchSnapshot()
})
