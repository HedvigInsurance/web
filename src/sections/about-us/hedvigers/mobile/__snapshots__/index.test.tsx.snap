// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.emotion-6 {
  padding: 20px 0;
}

.emotion-5 {
  margin: 20px;
  box-shadow: 0px 0 10px rgba(0,0,0,0.14);
  border-radius: 5px;
  overflow: hidden;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.emotion-1 {
  position: relative;
  overflow: hidden;
  height: 240px;
  width: 100%;
}

.emotion-0 {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  object-fit: cover;
}

.emotion-4 {
  background-color: white;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 20px;
}

.emotion-2 {
  font-size: 18px;
  font-family: So Ray;
}

.emotion-3 {
  font-size: 16px;
}

<Component
  teamtailorUsers={
    Array [
      Object {
        "name": "mock",
        "picture": Object {
          "large": "https://mockimage.com",
        },
        "title": "mock",
      },
    ]
  }
>
  <Styled(div)>
    <div
      className="emotion-6"
    >
      <MockMediaQuery
        query="(max-width: 700px)"
      >
        <MediaQuery
          query="(max-width: 700px)"
          values={
            Object {
              "height": 768,
              "width": 600,
            }
          }
        >
          <ReactSwipableView
            animateHeight={false}
            animateTransitions={true}
            axis="x"
            disableLazyLoading={false}
            disabled={false}
            enableMouseEvents={true}
            hysteresis={0.6}
            ignoreNativeScroll={false}
            index={0}
            resistance={true}
            slideStyle={
              Object {
                "display": "flex",
              }
            }
            springConfig={
              Object {
                "delay": "0s",
                "duration": "0.35s",
                "easeFunction": "cubic-bezier(0.15, 0.3, 0.25, 1)",
              }
            }
            style={
              Object {
                "padding": "0 15%",
              }
            }
            threshold={5}
          >
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseMove={[Function]}
              onMouseUp={[Function]}
              onScroll={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
              style={
                Object {
                  "overflowX": "hidden",
                  "padding": "0 15%",
                }
              }
            >
              <div
                className="react-swipeable-view-container"
                style={
                  Object {
                    "WebkitFlexDirection": "row",
                    "WebkitTransform": "translate(0%, 0)",
                    "WebkitTransition": "all 0s ease 0s",
                    "direction": "ltr",
                    "display": "flex",
                    "flexDirection": "row",
                    "height": null,
                    "transform": "translate(0%, 0)",
                    "transition": "all 0s ease 0s",
                    "willChange": "transform",
                  }
                }
              >
                <div
                  aria-hidden={false}
                  data-swipeable="true"
                  key=".$mock"
                  style={
                    Object {
                      "WebkitFlexShrink": 0,
                      "display": "flex",
                      "flexShrink": 0,
                      "overflow": "auto",
                      "width": "100%",
                    }
                  }
                >
                  <Styled(div)
                    key="mock"
                  >
                    <div
                      className="emotion-5"
                    >
                      <Styled(div)>
                        <div
                          className="emotion-1"
                        >
                          <Styled(img)
                            src="https://mockimage.com"
                          >
                            <img
                              className="emotion-0"
                              src="https://mockimage.com"
                            />
                          </Styled(img)>
                        </div>
                      </Styled(div)>
                      <Styled(div)>
                        <div
                          className="emotion-4"
                        >
                          <Styled(span)>
                            <span
                              className="emotion-2"
                            >
                              mock
                            </span>
                          </Styled(span)>
                          <Styled(span)>
                            <span
                              className="emotion-3"
                            >
                              mock
                            </span>
                          </Styled(span)>
                        </div>
                      </Styled(div)>
                    </div>
                  </Styled(div)>
                </div>
              </div>
            </div>
          </ReactSwipableView>
        </MediaQuery>
      </MockMediaQuery>
      <MockMediaQuery
        query="(min-width: 700px)"
      >
        <MediaQuery
          query="(min-width: 700px)"
          values={
            Object {
              "height": 768,
              "width": 600,
            }
          }
        />
      </MockMediaQuery>
    </div>
  </Styled(div)>
</Component>
`;
